### ISSUE TITLE
The link for Linux is broken on the Readme page

### ISSUE THREAD

**Issue by yanngit** [andreaskoch/allmark#23]
The command provided for Linux is not working properly ...

**Comment by matchavez** [comment]
Yep. Can't install right now. Frustrating. I was excited to try this...

**Comment by shadiakiki1986** [comment]
You can download the binaries from the github releases page [here](https://github.com/andreaskoch/allmark/releases)

**Comment by matchavez** [comment]
That's not the issue. See issue #12 because while you can download it, there IS a bug.
